diff --git a/node_modules/@strapi/admin/admin/src/content-manager/components/Wysiwyg/index.js b/node_modules/@strapi/admin/admin/src/content-manager/components/Wysiwyg/index.js
index cdf094b..a6f29bf 100644
--- a/node_modules/@strapi/admin/admin/src/content-manager/components/Wysiwyg/index.js
+++ b/node_modules/@strapi/admin/admin/src/content-manager/components/Wysiwyg/index.js
@@ -102,7 +102,9 @@ const Wysiwyg = ({
 
   const handleSelectAssets = (files) => {
     const formattedFiles = files.map((f) => ({
-      alt: f.alternativeText || f.name,
+      // This adds caption into alt property so we can use both on the frontend.
+      // Inserted image in markdown will look like this: ![alt||caption](url)
+      alt: `${f.alternativeText}||${f.caption}` || f.name,
       url: prefixFileUrlWithBackendUrl(f.url),
       mime: f.mime,
     }));
