diff --git a/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js b/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
index 17a52c4..9c1210f 100644
--- a/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
+++ b/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
@@ -12584,7 +12584,7 @@ const Wysiwyg = React__namespace.forwardRef(
     };
     const handleSelectAssets = (files) => {
       const formattedFiles = files.map((f) => ({
-        alt: f.alternativeText || f.name,
+        alt: `${f.alternativeText || ''}||${f.caption || ''}`,
         url: helperPlugin.prefixFileUrlWithBackendUrl(f.url),
         mime: f.mime
       }));
diff --git a/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs b/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
index 95a5860..2c5e7a1 100644
--- a/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
+++ b/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
@@ -12524,7 +12524,7 @@ const Wysiwyg = React.forwardRef(
     };
     const handleSelectAssets = (files) => {
       const formattedFiles = files.map((f) => ({
-        alt: f.alternativeText || f.name,
+        alt: `${f.alternativeText || ''}||${f.caption || ''}`,
         url: prefixFileUrlWithBackendUrl(f.url),
         mime: f.mime
       }));
