diff --git a/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js b/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
index 17a52c4..06f5c96 100644
--- a/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
+++ b/node_modules/@strapi/admin/dist/_chunks/index-dk76Bb5X.js
@@ -758,7 +758,7 @@ const SETTINGS_LINKS_CE = () => ({
     },
     // If the Enterprise/Cloud feature is not enabled and if the config doesn't disable it, we promote the Enterprise/Cloud feature by displaying them in the settings menu.
     // Disable this by adding "promoteEE: false" to your `./config/admin.js` file
-    ...!window.strapi.features.isEnabled(window.strapi.features.SSO) && window.strapi?.flags?.promoteEE ? [
+    ...(!window.strapi.features.isEnabled(window.strapi.features.SSO) && window.strapi?.flags?.promoteEE ? [
       {
         intlLabel: { id: "Settings.sso.title", defaultMessage: "Single Sign-On" },
         to: "/settings/purchase-single-sign-on",
@@ -766,8 +766,8 @@ const SETTINGS_LINKS_CE = () => ({
         lockIcon: true
         // TODO: to replace with another name in v5
       }
-    ] : [],
-    ...!window.strapi.features.isEnabled(window.strapi.features.REVIEW_WORKFLOWS) && window.strapi?.flags?.promoteEE ? [
+    ] : []),
+    ...(!window.strapi.features.isEnabled(window.strapi.features.REVIEW_WORKFLOWS) && window.strapi?.flags?.promoteEE ? [
       {
         intlLabel: {
           id: "Settings.review-workflows.page.title",
@@ -778,7 +778,7 @@ const SETTINGS_LINKS_CE = () => ({
         lockIcon: true
         // TODO: to replace with another name in v5
       }
-    ] : []
+    ] : [])
   ],
   admin: [
     {
@@ -792,7 +792,7 @@ const SETTINGS_LINKS_CE = () => ({
       to: "/settings/users?pageSize=10&page=1&sort=firstname",
       id: "users"
     },
-    ...!window.strapi.features.isEnabled(window.strapi.features.AUDIT_LOGS) && window.strapi?.flags?.promoteEE ? [
+    ...(!window.strapi.features.isEnabled(window.strapi.features.AUDIT_LOGS) && window.strapi?.flags?.promoteEE ? [
       {
         intlLabel: { id: "global.auditLogs", defaultMessage: "Audit Logs" },
         to: "/settings/purchase-audit-logs",
@@ -800,7 +800,7 @@ const SETTINGS_LINKS_CE = () => ({
         lockIcon: true
         // TODO: to replace with another name in v5
       }
-    ] : []
+    ] : [])
   ]
 });
 const admin = adminApi.injectEndpoints({
@@ -4284,7 +4284,7 @@ class CustomFields {
         isValidObjectKey.test(name2),
         `Custom field name: '${name2}' is not a valid object key`
       );
-      const allFormOptions = [...options2?.base || [], ...options2?.advanced || []];
+      const allFormOptions = [...(options2?.base || []), ...(options2?.advanced || [])];
       if (allFormOptions.length) {
         const optionPathValidations = allFormOptions.reduce(optionsValidationReducer, []);
         optionPathValidations.forEach(({ isValidOptionPath, errorMessage }) => {
@@ -5396,7 +5396,7 @@ const useRelation = (cacheKey = [], { relation, search }) => {
       try {
         const { data: data2 } = await get2(relation?.endpoint, {
           params: {
-            ...relation.pageParams ?? {},
+            ...(relation.pageParams ?? {}),
             page: pageParam
           }
         });
@@ -5463,7 +5463,7 @@ const useRelation = (cacheKey = [], { relation, search }) => {
       try {
         const { data: data2 } = await get2(search.endpoint, {
           params: {
-            ...search.pageParams ?? {},
+            ...(search.pageParams ?? {}),
             ...searchParams,
             page: pageParam
           }
@@ -16824,7 +16824,7 @@ const usersService = adminApi.injectEndpoints({
           return [{ type: "User", id: arg.id }];
         } else {
           return [
-            ...res?.users.map(({ id }) => ({ type: "User", id })) ?? [],
+            ...(res?.users.map(({ id }) => ({ type: "User", id })) ?? []),
             { type: "User", id: "LIST" }
           ];
         }
@@ -16875,7 +16875,7 @@ const usersService = adminApi.injectEndpoints({
           return [{ type: "Role", id: arg.id }];
         } else {
           return [
-            ...res?.map(({ id }) => ({ type: "Role", id })) ?? [],
+            ...(res?.map(({ id }) => ({ type: "Role", id })) ?? []),
             { type: "Role", id: "LIST" }
           ];
         }
@@ -17546,7 +17546,7 @@ const useAllowedAttributes = (contentType, slug) => {
     "id",
     ...allowedAttributes,
     ...DEFAULT_ALLOWED_FILTERS,
-    ...canReadAdminUsers ? CREATOR_FIELDS : []
+    ...(canReadAdminUsers ? CREATOR_FIELDS : [])
   ];
 };
 const { INJECT_COLUMN_IN_TABLE: INJECT_COLUMN_IN_TABLE$1 } = HOOKS;
@@ -17708,7 +17708,7 @@ const ListViewPage = ({
               ...eeFilter.metadatas,
               // the stage filter needs the current content-type uid to fetch
               // the list of stages that can be assigned to this content-type
-              ...eeFilter.name === "strapi_stage" ? { uid: contentType.uid } : {},
+              ...(eeFilter.name === "strapi_stage" ? { uid: contentType.uid } : {}),
               // translate the filter label
               label: formatMessage(eeFilter.metadatas.label),
               // `options` allows the filter-tag to render the displayname
@@ -19035,7 +19035,7 @@ class StrapiApp {
     if (this.customConfigurations?.locales) {
       this.configurations.locales = [
         "en",
-        ...this.customConfigurations.locales?.filter((loc) => loc !== "en") || []
+        ...(this.customConfigurations.locales?.filter((loc) => loc !== "en") || [])
       ];
     }
     if (this.customConfigurations?.auth?.logo) {
@@ -19182,8 +19182,8 @@ class StrapiApp {
     const translations = this.configurations.locales.reduce((acc, current) => {
       acc[current] = {
         ...adminTranslations[current],
-        ...mergedTrads[current] || {},
-        ...this.customConfigurations?.translations?.[current] ?? {}
+        ...(mergedTrads[current] || {}),
+        ...(this.customConfigurations?.translations?.[current] ?? {})
       };
       return acc;
     }, {});
diff --git a/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs b/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
index 95a5860..0f33be1 100644
--- a/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
+++ b/node_modules/@strapi/admin/dist/_chunks/index-uCIlbOdV.mjs
@@ -12524,7 +12524,7 @@ const Wysiwyg = React.forwardRef(
     };
     const handleSelectAssets = (files) => {
       const formattedFiles = files.map((f) => ({
-        alt: f.alternativeText || f.name,
+        alt: `${f.alternativeText}||${f.caption}`,
         url: prefixFileUrlWithBackendUrl(f.url),
         mime: f.mime
       }));
