diff --git a/node_modules/strapi-plugin-navigation/server/content-types/audience/schema.js b/node_modules/strapi-plugin-navigation/server/content-types/audience/schema.js
index db9af02..e4914bf 100644
--- a/node_modules/strapi-plugin-navigation/server/content-types/audience/schema.js
+++ b/node_modules/strapi-plugin-navigation/server/content-types/audience/schema.js
@@ -12,6 +12,11 @@ exports.default = {
         increments: true,
         comment: "Audience"
     },
+    pluginOptions: {
+        "content-manager": {
+            "visible": false
+        }
+    },
     attributes: {
         name: {
             type: "string",
