diff --git a/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/component.js b/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/component.js
index 62fe5ae..82e3282 100644
--- a/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/component.js
+++ b/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/component.js
@@ -23,7 +23,9 @@ const MediaLibComponent = ({isOpen, onChange, onToggle}) => {
       height: f.height,
       size: f.size,
       mime: f.mime,
-      formats: f.formats
+      formats: f.formats,
+      // Our own customization, add full strapi image properties to display images with <MImage>
+      strapiImage: f
     }));
     onChange(formattedFiles);
   };
diff --git a/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/utils.js b/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/utils.js
index c720cae..4bb9916 100755
--- a/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/utils.js
+++ b/node_modules/strapi-plugin-react-editorjs/admin/src/components/medialib/utils.js
@@ -26,6 +26,8 @@ export const changeFunc = ({indexStateSetter, editor, data, index}) => {
         size: entry.size,
         alt: entry.alt,
         formats: entry.formats,
+        // Our own customization, add full strapi image properties to display images with <MImage>
+        strapiImage: entry.strapiImage
       },
       caption: "",
       withBorder: false,
diff --git a/node_modules/strapi-plugin-react-editorjs/admin/src/config/customTools.js b/node_modules/strapi-plugin-react-editorjs/admin/src/config/customTools.js
index d725af8..6182b40 100644
--- a/node_modules/strapi-plugin-react-editorjs/admin/src/config/customTools.js
+++ b/node_modules/strapi-plugin-react-editorjs/admin/src/config/customTools.js
@@ -67,4 +67,12 @@ const customTools = {
   inlineCode: InlineCode,
 }
 
-export default customTools
+const customToolsFixed = {
+  table: customTools.table,
+  header: customTools.header,
+  delimiter: customTools.delimiter,
+  list: customTools.list,
+  quote: customTools.quote,
+}
+
+export default customToolsFixed
